modules/consensus/synchronize_test.go

```diff
@@ -108,7 +107,6 @@ func TestBlockHistory(t *testing.T) {
 	if testing.Short() {
 		t.SkipNow()
 	}
-	t.Parallel()
```

Comment:
It was causing my laptop to time out a bunch as well, I think that something with the I/O caching and screwing up Bolt would be related. Or maybe the scheduler was just having trouble.

Post this PR, we should continue to pay attention to the timeouts. My 8-core desktop + SSD definitely speeds up when we have every test running t.Parallel, but travis is a lot more resource constrained. It might we worth only subjecting a test to t.Parallel if it spends a significant amount of time idling.